### DISCUSSION TITLE
blog-2022-03-16-kotlin-exposed-postgresql-timestamp-with-time-zone

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#12] [Fan Comments]
# blog-2022-03-16-kotlin-exposed-postgresql-timestamp-with-time-zone

Exposed's exposed-java-time (and exposed-kotlin-datetime) module has an issue: In PostgreSQL it uses TIMESTAMP WITHOUT TIME ZONE and always saves the timestamp using the system's current time zone (ZoneId.systemDefault()).

https://mrpowergamerbr.com/us/blog/2022-03-16-kotlin-exposed-postgresql-timestamp-with-time-zone/

**Comment by schrek1** [comment]
very useful, it is bad that timestamp with timezone is not implemented out of the box..
